known_good.json

```
@@ -5,14 +5,14 @@
       "site" : "github",
       "subrepo" : "KhronosGroup/SPIRV-Tools",
       "subdir" : "External/spirv-tools",
-      "commit" : "eed5c76a57bb965f2e1b56d1dc40b50910b5ec1d"
+      "commit" : "58dc37ea6af7ec09f32f7b13ff60071e3ba2bd24"
```

```suggestion
      "commit" : "9e377b0f974bd1f048e20dda1d52da900b3246cb"
```
(this is the commit that actually adds cull mask support)